wfe2/wfe.go

```diff
@@ -233,6 +233,13 @@ func (wfe *WebFrontEndImpl) HandleFunc(mux *http.ServeMux, pattern string, h web
 				logEvent.Slug = request.URL.Path
 				beeline.AddFieldToTrace(ctx, "slug", request.URL.Path)
 			}
+			if !features.Enabled(features.OldTLSInbound) {
+				tls := request.Header.Get("TLS-Version")
+				if tls == "TLSv1" || tls == "TLSv1.1" {
```

Comment:
I think you can use the following here:

```go
if request.TLS.Version < tls.VersionTLS12 {
    ...
}
```

Reply:
I mean for line, `238`. I'm afraid I was a little unclear with my highlighting.

Reply:
This doesn't work, because the request as we receive it isn't actually TLS; it's plaintext HTTP, received from Nginx. Nginx tells us what TLS version it saw via the TLS-Version header (which we set specially in our config).